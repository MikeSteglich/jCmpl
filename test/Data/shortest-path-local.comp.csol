<?xml version = "1.0" encoding="UTF-8" standalone="yes"?>
<CmplSolutions version="1.1">
   <general>
       <instanceName>shortest-path.cmpl</instanceName>
       <nrOfVariables>22</nrOfVariables>
       <nrOfConstraints>7</nrOfConstraints>
       <objectiveName>line_1</objectiveName>
       <objectiveSense>min</objectiveSense>
       <nrOfSolutions>1</nrOfSolutions>
       <solverName>CBC</solverName>
       <solverMsg>normal</solverMsg>
       <variablesDisplayOptions>nonZeroVariables (all)</variablesDisplayOptions>
       <constraintsDisplayOptions>nonZeroConstraints (all)</constraintsDisplayOptions>
   </general>
   <solution idx="0" status="optimal" value="22">
       <variables>
           <variable idx="1" name="x[1,4]" type="C" activity="1" lowerBound="0" upperBound="inf" marginal="0"/>
           <variable idx="11" name="x[4,6]" type="C" activity="1" lowerBound="0" upperBound="inf" marginal="0"/>
           <variable idx="19" name="x[6,7]" type="C" activity="1" lowerBound="0" upperBound="inf" marginal="0"/>
       </variables>
       <linearConstraints> 
           <constraint idx="0" name="node[1]" type="E" activity="1" lowerBound="1" upperBound="1" marginal="8"/>
           <constraint idx="6" name="node[7]" type="E" activity="-1" lowerBound="-1" upperBound="-1" marginal="-14"/>
       </linearConstraints>
   </solution>
</CmplSolutions>
