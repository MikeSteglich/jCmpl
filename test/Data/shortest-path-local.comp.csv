CMPL csv export

Problem;shortest-path.cmpl
Nr. of variables;22
Nr. of constraints;7
Objective name;line_1
Solver name;CBC
Display variables;nonZeroVariables (all)
Display constraints;nonZeroConstraints (all)

Objective status;optimal
Objective value;22.000000;(min!)
Variables
Name;Type;Activity;LowerBound;UpperBound;Marginal
x[1,4];C;1.000000;0.000000;Infinity;0.000000
x[4,6];C;1.000000;0.000000;Infinity;0.000000
x[6,7];C;1.000000;0.000000;Infinity;0.000000
Constraints
Name;Type;Activity;LowerBound;UpperBound;Marginal
node[1];E;1.000000;1.000000;1.000000;8.000000
node[7];E;-1.000000;-1.000000;-1.000000;-14.000000
