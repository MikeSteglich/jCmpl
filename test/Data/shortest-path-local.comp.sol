---------------------------------------------------------------------------------------------------------
Problem              shortest-path.cmpl
Nr. of variables     22
Nr. of constraints   7
Objective name       line_1
Solver name          CBC
Display variables    nonZeroVariables (all)
Display vonstraints  nonZeroConstraints (all)
---------------------------------------------------------------------------------------------------------

Objective status     optimal
Objective value      22.00                (min!)

Variables           
Name                 Type            Activity          LowerBound          UpperBound            Marginal
---------------------------------------------------------------------------------------------------------
x[1,4]                  C                1.00                0.00            Infinity                0.00
x[4,6]                  C                1.00                0.00            Infinity                0.00
x[6,7]                  C                1.00                0.00            Infinity                0.00
---------------------------------------------------------------------------------------------------------

Constraints         
Name                 Type            Activity          LowerBound          UpperBound            Marginal
---------------------------------------------------------------------------------------------------------
node[1]                 E                1.00                1.00                1.00                8.00
node[7]                 E               -1.00               -1.00               -1.00              -14.00
---------------------------------------------------------------------------------------------------------
