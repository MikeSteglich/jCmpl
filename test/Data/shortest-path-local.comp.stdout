Objective value     :22.0
x[1,4] 1.0
x[4,6] 1.0
x[6,7] 1.0
